tests/integration/testecbug.nim

```diff
@@ -10,7 +10,7 @@ marketplacesuite "EC bug":
     NodeConfigs(
       # Uncomment to start Hardhat automatically, typically so logs can be
       # inspected locally
-      hardhat: HardhatConfig().withLogFile().some,
+      hardhat: HardhatConfig.none,
```

Comment:
Probably just better to comment out, as it says above?

Reply:
I copied it from existing configuration in other test. If you want I can try with commenting it out.
https://github.com/codex-storage/nim-codex/blob/5b131611acb632e0ef4554996f9b774b534ff91b/tests/integration/testmarketplace.nim#L89

Reply:
Ok, I guess either way works. But when we enable this, we need to disable to starting hardhat from the CI, because I confirmed that its whats causing the failed tests - at least, locally.

Reply:
I just deleted the whole thing. The integration suite doesn't work with this right now, so I don't think it makes sense to keep it here.